Test/Utils/ComputeNode/Initialize.ps1

```diff
@@ -9,14 +9,45 @@ function Initialize-ComputeNode {
     Param (
         [Parameter(Mandatory=$true)] [PSSessionT] $Session,
         [Parameter(Mandatory=$true)] [TestenvConfigs] $Configs,
-        [Parameter(Mandatory=$true)] [Network[]] $Networks
+        [Parameter(Mandatory=$false)] [bool] $PrepareEnv=$true
```

Comment:
This goes for `Clear-ComputeNode` as well.

Could you explain why we pass `$PrepareEnv` to this function and then we decide to do nothing? Why not put this logic in clients for this function? E.g. instead of

```
Clear-ComputeNode -Session $Session -SystemConfig $MultiNode.Configs.System -PrepareEnv $PrepareEnv
```

have

```
if($PrepareEnv) { Clear-ComputeNode -Session $Session -SystemConfig $MultiNode.Configs.System }
```

Reply:
Well, I thought it would make tests more readable, but I can change if you'd prefer it other way.